### ISSUE TITLE
Unclosed Socket

### ISSUE THREAD

**Issue by Phill93** [GhostofGoes/getmac#42]
**Describe the bug**
A socket is opened in line 143 of the [getmac.py](getmac.py) file. However, this socket is not closed again.

This leads to a "ResourceWarning".

```
/Users/xxxx/projects/netmap/netmap/network.py:18: ResourceWarning: unclosed <socket.socket fd=3, family=AddressFamily.AF_INET, type=SocketKind.SOCK_DGRAM, proto=0, laddr=('0.0.0.0', 50240)>
  self.mac_address = get_mac_address(ip=str(self.ip_address.ip))
Object allocated at (most recent call last):
  File "/Users/xxxx/projects/netmap/venv/lib/python3.7/site-packages/getmac/getmac.py", lineno 143
    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
```

**To Reproduce**
Run a unittest that contains a get_mac_address call that requests a mac through a ip.

**Expected behavior**
No Warning

**System info**
(please complete the following information):
 - OS name: macOS 
 - OS Version: 10.14.4
 - Python version: 3.7.3 x64
 - getmac version: 0.8.0

**Comment by GhostofGoes** [comment]
Thanks for the report, and apologies for the delay in responding. This is definitely a issue and one I'm surprised has been overlooked until now. I've pushed a potential fix to master, please give it a try. You can install like so: `pip install https://github.com/ghostofgoes/getmac/archive/master.tar.gz` (that URL works in a requirements.txt/setup.py/etc. as well).

If that fixes it I'll get a point release cut and released as soon as I can.

**Comment by Phill93** [comment]
The fix works. Thanks

**Comment by GhostofGoes** [comment]
Released on GitHub and uploaded to PyPI as 0.8.1.
